src/controllers/post.controller.ts

```diff
@@ -0,0 +1,28 @@
+// 이곳은 req, res만을 관리함 (controller)
+// req로부터 params 및 body를 꺼내서 service 영역으로 전달 & service로부터 데이터를 받아옴
+import {Request, Response, NextFunction} from 'express';
+import * as postService from '../services/post.service';
+
+export async function getPosts(
+    req: Request, res: Response, next: NextFunction) {
+  try {
+    const posts = await postService.getPosts();
+    res.status(200).send(posts);
+  } catch (err) {
+    res.status(500).send(`Error while find posts`);
```

Comment:
```suggestion
    res.status(HTTP_ERROS.OK).send(posts);
  } catch (err) {
    res.status(HTTP_ERROS.INTERNAL_SERVER_ERROR).send(`Error while find posts`);
```

HTTP 상태코드는 이런식으로 상수화시키는건 어떠신가여
상태코드를 다 외우고 있는게 아니라서 이런식으로 상수화시켜 놓으면 의미가 딱 보이지 않을까 싶어서 지나가다 리뷰 남깁니다 펭긴님ㅎㅎㅎ

하실거면 상태코드는 [https://developer.mozilla.org/ko/docs/Web/HTTP/Status](https://developer.mozilla.org/ko/docs/Web/HTTP/Status) 여기 참고해서 네이밍하면 될 듯 합니다ㅎㅎ

Reply:
넵 감사합니다. 이 부분은 error handler 작업과 함께 다음 브랜치에서 반영하겠습니다~